docs/js-core-1/week-2/lesson.md

```diff
@@ -108,46 +41,40 @@ let codeYourFutureIsGreat = true;
 let thisIsATerribleClass = false;
 ```
 
-### Exercise (10 mins)
+We can use boolean values to make decisions in our code based on certain conditions, as we shall later on.
 
-1. In a node REPL, what is the `typeof` a `true` or `false`?
-2. Pair up and correct the following function so that the output returns `"You've given me a bool, thanks!"`
+## Comparison Operators
 
-```js
-function boolChecker(bool) {
-  if (typeof bool === ) {
-    return "You've given me a bool, thanks!";
-  }
+We can also create an **expression** that evaluates to a `boolean` value - that is, either `true` or `false`. This is possible when we use  **comparison operators**  such as `===`. This comparison operator will check if two values are the same.
 
-  return "No bool, not cool.";
-}
+For example, 
 
-boolChecker(true);
-```
-
-3. As a class, can you step through the function and explain what each line does?
+`42 === 42` will evaluate to `true`.
+You can check this by logging the expression `42 === 42` in the console.
```

Comment:
I wonder if we should change "console" to "Node REPL"?

I think this could be interpreted as adding `console.log(42 === 42)` to a file and running it. Which isn't wrong per-se, but I think the intention is to run a quick check in the repl. Adding more logging to their working file may also get confusing.